code/game/objects/items/apc_frame.dm

```diff
@@ -88,10 +88,10 @@
 	inverse = 1
 
 
-/obj/item/wallframe/apc/try_build(turf/on_wall, user)
+/obj/item/wallframe/apc/try_build(turf/on_wall, mob/user)
 	if(!..())
 		return
-	var/turf/T = get_turf(on_wall) //the user is not where it needs to be.
+	var/turf/T = get_turf(user.loc) //get the turf the user is standing on, not where it's being placed.
```

Comment:
Default back to the turf if the user or their loc is null

Reply:
> Default back to the turf if the user or their loc is null

no